hadoop-ozone/dist/src/main/license/jar-report.txt

```diff
@@ -145,13 +146,17 @@ share/ozone/lib/jetty-util-ajax.jar
 share/ozone/lib/jetty-util.jar
 share/ozone/lib/jetty-webapp.jar
 share/ozone/lib/jetty-xml.jar
+share/ozone/lib/jgraph.jar
+share/ozone/lib/jgrapht-core.jar
+share/ozone/lib/jgrapht-ext.jar
+share/ozone/lib/jgraphx.jar
```

Comment:
Please restrict changes in `jar-report.txt` to addition of the new lines, avoid reordering existing lines.

See discussion here: https://github.com/apache/ozone/pull/4985#discussion_r1243165718

Reply:
Initially I tired to restrict it but it failed because some transitive dependency was not there. So I used the auto generated one.

Updated it and manually added only necessary changes.

Thanks for pointing it out.